config/_default/config.toml

```diff
@@ -0,0 +1,8 @@
+# baseURL = "https://florisboard.org/"
+baseURL = "https://f.delthia.com/"
```

Comment:
Maybe this could be made to work by having a different action when it is in a fork or on another branch, I haven't much experience with that, but if I find a simple way to make it work like that, maybe it would be easier if people contribute translations.

Reply:
Hmm I don't really know if an action can be configured to run with different params tbh, and even then how do you configure this param without having to commit it to the fork?